### DISCUSSION TITLE
#include errors detected. Please update your includePath. Squiggles are disabled for this translation unit (D:\C++ Tutorial\helloworld.cpp).

### DISCUSSION THREAD

**Discussion by amans630640** [microsoft/vscode-cpptools#8640] [Q&A] [answered]
![Screenshot (132)](https://user-images.githubusercontent.com/95912594/148444024-bcb12b32-a1ad-417c-9b9c-d9fa9c2ec7f4.png)

**Comment by amans630640** [comment]
please givee me a solution to fix this problem in my vs code

**Comment by amans630640** [comment, marked as answer]
![Screenshot (133)](https://user-images.githubusercontent.com/95912594/148444424-59dcb1e5-34b6-4d38-b19d-f02f2c78d043.png)

This is everytime showing in the problem section

**Reply by bobbrow** [reply]
Do you have a compiler installed?  We use that to locate the C++ library headers.  When this happens it usually means that we couldn't find your compiler and you either need to install one or tell us where it is.  A guide that may help you can be found here: https://code.visualstudio.com/docs/cpp/config-mingw

**Reply by amans630640** [reply]
thank you so much sir..you are guiding me
sir i have already installed the compiler migw-w64 . sir i have tried all the means to fix this problem but i couldn't fix it

**Reply by amans630640** [reply]
![Screenshot (134)](https://user-images.githubusercontent.com/95912594/148495954-e3768141-eea9-44d1-8d2d-db6e3aedd6dd.png)
sir what to write in ompiler path

**Reply by amans630640** [reply]
Thank you very much sir for your help @bobbrow 
I have successfully fixed the problem of include path in vs code with this you tube video
 https://youtu.be/mWTjF7EWZD4

**Reply by bobbrow** [reply]
I saw your video, but the solution you presented is not the preferred one.  You should set the `compilerPath` to the path of your compiler, not the `includePath`.  We will automatically add the `includePath` of the system headers and the corresponding default `defines` of the compiler if you set `compilerPath`.  This will get you more correct IntelliSense results when you do more than just simple `printf` or `cout`.

**Reply by amans630640** [reply]
sir i am unable to perform what you have mentioned in the reply..sir if you , agree please take my system remote with quick assist and fix the issue as you have mentioned..i shall be highly obliged

**Reply by bobbrow** [reply]
![image](https://user-images.githubusercontent.com/12818240/148581659-cd7242c6-41dd-4554-bf38-e6fd40eb9dae.png)

**Reply by amans630640** [reply]
Done sir, Thank you so much for this image,, Without that image i wouldn't have been able to correctly set the compiler path..
Thanks alot sir @bobbrow
